### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [leihl/leihl.github.io#1]
The user @carlsonsantana validate your site template "https://leihl.github.io" and found these accessibility errors:
* Page https://leihl.github.io/:
  * Issue *7554a252-54ca-450b-b881-f267c20275c7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
    * **Message**: Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
    * **Selector**: `html > body > img`
    * **Context**: `<img src="/img/common/apple-touch-icon.png" width="0" height="0">`
  * Issue *a9a91518-e4d1-488c-91e1-78c985882169*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
    * **Message**: Iframe element requires a non-empty title attribute that identifies the frame.
    * **Selector**: `html > body > footer > div > div > div > p > iframe`
    * **Context**: `<iframe style="margin-left: 2px; margin-bottom:-5px;" frameborder="0" scrolling="0" width="100px" height="20px" src="https://ghbtns.com/github-btn.html?user=&amp;repo=.github.io&amp;type=star&amp;count=true">
                    </iframe>`
  * Issue *d77531bf-e531-4e5a-8837-97647ca0fc56*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.48:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > footer > div > div > div > p`
    * **Context**: `<p class="copyright text-muted">
                    Copyright ...</p>`
  * Issue *4f31394f-0e14-4f4c-8187-a9ce1c94694e*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.84:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(2) > ul > li:nth-child(2) > a`
    * **Context**: `<a href="http://.com">博客园</a>`
  * Issue *38175452-1406-43b3-ae10-297c0a221afc*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.84:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(2) > ul > li:nth-child(1) > a`
    * **Context**: `<a href="https://blog.csdn.net/leisure_chn">CSDN</a>`
  * Issue *b0c1d9e3-5e35-4b81-ab50-d098845989c8*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(2) > h5`
    * **Context**: `<h5>FRIENDS</h5>`
  * Issue *a6cb5add-ae63-4db8-aa5a-c1bbe2c6d690*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.84:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(2) > section:nth-child(2) > div > p`
    * **Context**: `<p>Goals determine what you going ...</p>`
  * Issue *549edb25-c932-4451-9a05-890261825268*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
    * **Message**: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
    * **Selector**: `html > body > div > div > div:nth-child(2) > section:nth-child(2) > div > a`
    * **Context**: `<a href="/about">
                              ...</a>`
  * Issue *4b162d6b-463d-45b6-94cc-28072e8f71ed*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(2) > section:nth-child(2) > h5 > a`
    * **Context**: `<a href="/about/">ABOUT ME</a>`
  * Issue *5be2554e-22af-4098-9d1e-8bec7e77f54a*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(2) > section:nth-child(1) > h5 > a`
    * **Context**: `<a href="/tags/">FEATURED TAGS</a>`
  * Issue *013c098c-a6fc-4043-af34-cf5fb004eec7*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(1) > div:nth-child(5) > p`
    * **Context**: `<p class="post-meta">
        Posted by leisure blog...</p>`
  * Issue *fe8b716d-ecf1-4384-b47f-99350e26f43c*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.52:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(1) > div:nth-child(5) > a > div`
    * **Context**: `<div class="post-content-preview">
            转载请注明出处：https://le...</div>`
  * Issue *db88260c-a96c-4fe8-bd50-97455671ce10*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(1) > div:nth-child(3) > p`
    * **Context**: `<p class="post-meta">
        Posted by leisure blog...</p>`
  * Issue *9a8e9782-231f-4b2f-b3fd-364aa0a6719a*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.52:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(1) > div:nth-child(3) > a > div`
    * **Context**: `<div class="post-content-preview">
            一、问题描述
调用ffmpeg的av...</div>`
  * Issue *eb571abf-a3ec-41ef-8cf9-24b3ef2f67cc*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.95:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(1) > div:nth-child(1) > p`
    * **Context**: `<p class="post-meta">
        Posted by leisure blog...</p>`
  * Issue *1e048f41-f17c-47b9-a414-bcf14d00d5f2*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.52:1. Recommendation: change text colour to #767676.
    * **Selector**: `html > body > div > div > div:nth-child(1) > div:nth-child(1) > a > div`
    * **Context**: `<div class="post-content-preview">
            
  
    
      日期
...</div>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/01979198-4640-420b-a263-c58b3ed09ed0/).
